mongo_connector/doc_managers/doc_manager_base.py

```diff
@@ -20,6 +20,10 @@
 class DocManagerBase(object):
     """Base class for all DocManager implementations."""
 
+    version_info = tuple()
+    version = 'unkown'
```

Comment:
This is somewhat counter to convention. Usually, the version is called `__version__` and is defined in a central module like `__init__.py`. Having it here in the class is somewhat strange. For example, the Elastic doc manager may have two classes, the DocManager and the BulkBuffer, but both of these classes conceptually have the same version.

Perhaps the convention could be to have a `__version__` defined at the module level. We can check for that when importing the doc manager in connector.py. Also, rather than having the string 'unknown' as a class member, we can just say 'unknown' if `not hasattr(module, '__version__')`.